package.json

```diff
@@ -8,19 +8,24 @@
     "format": "prettier --write \"**/*.{js,cjs,mjs,ts,tsx,md,json}\" --ignore-path .gitignore",
     "lint": "next lint",
     "start": "next start",
-    "prepare": "husky install"
+    "prepare": "husky install && chmod ug+x ./.husky/*"
   },
   "lint-staged": {
     "*.{js,cjs,mjs,ts,tsx,md,json}": "prettier --write"
   },
   "dependencies": {
+    "emoji-picker-react": "^4.4.8",
+    "marked-react": "^1.3.2",
     "next": "13.2.4",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "react-icons": "^4.8.0",
+    "styled-components": "^5.3.9"
```

Comment:
styled components you mean? it's pretty straightforward and the idea i had was for creating a couple of reusable components to keep the styling consistent, but working with it would be optional still